src/main/java/com/jwplayer/southpaw/Southpaw.java

```diff
@@ -144,20 +144,12 @@
     protected static class Config {
         public static final String BACKUP_TIME_S_CONFIG = "backup.time.s";
         public static final int BACKUP_TIME_S_DEFAULT = 1800;
-        public static final String BACKUP_TIME_S_DOC = "Time interval (roughly) between backups";
```

Comment:
Ok with removing these unused vars, but can we instead convert these to docstrings for the config values

Reply:
moved these values to their respective fields